tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -521,39 +524,43 @@ def run_test(self, test):
                 urls[i] = node.url
 
             is_pass, page_idx = self.check_pass(hashes, screenshots, urls, relation, fuzzy)
+            log_data = [
+                {"url": urls[0], "screenshot": screenshots[0][page_idx],
+                 "hash": hashes[0][page_idx]},
+                relation,
+                {"url": urls[1], "screenshot": screenshots[1][page_idx],
+                 "hash": hashes[1][page_idx]}
+            ]
+
             if is_pass:
                 fuzzy = self.get_fuzzy(test, nodes, relation)
                 if nodes[1].references:
                     stack.extend(list(((nodes[1], item[0]), item[1])
                                       for item in reversed(nodes[1].references)))
                 else:
+                    test_result = {"status": "PASS", "message": None}
+                    if (self.reftest_screenshot == "always" or
+                            self.reftest_screenshot == "unexpected" and
```

Comment:
Style in this code is to indent this to align with the line above, not to add an extra level of indent.

Reply:
Done